packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,39 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
+  const [unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);
+  const [showChat, setShowChat] = useState(false);
+  const [showPeople, setShowPeople] = useState(false);
+
   meetingAdapter.onStateChange((newState) => {
     setCurrentPage(newState.page);
     setCurrentMeetingState(newState.meeting?.state);
   });
 
-  const [showChat, setShowChat] = useState(false);
-  const [showPeople, setShowPeople] = useState(false);
+  useEffect(() => {
+    const incrementUnreadtChatMessagesCount = (event: { message: ChatMessage }): void => {
+      if (!showChat && event.message.senderDisplayName !== '') {
+        if (event.message.type === 'text' || event.message.type === 'html') {
```

Comment:
I'm a bit worried about this check being repeated multiple times in our codebase. Can we export an `@internal` helper for this from `acs-ui-common`?

https://github.com/Azure/communication-ui-library/blob/f87bf0c6588a4fff9469a130d0492ce429a57779/packages/chat-component-bindings/src/messageThreadSelector.ts#L29
https://github.com/Azure/communication-ui-library/blob/cf149e82f77ce8bd7c4707b1bfc84bbfd9c9c2df/packages/chat-stateful-client/src/EventSubscriber.ts#L55

Reply:
made a helper function in another PR: https://github.com/Azure/communication-ui-library/pull/1411